.github/hooks/commit-msg

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+"""The commit-msg Git hook tp check the commit message."""
+
+import sys
+from enum import Enum
+
+
+class BColors(str, Enum):
+    """A Enum for colors using ANSI escape sequences.
+    Reference:
+    - https://stackoverflow.com/questions/287871
+    """
+    OK       = '\u001b[32m'
+    INFO     = '\u001b[33m'
+    WARNING  = '\u001b[34m'
+    ERROR    = '\u001b[35m'
+    BOLD     = '\u001b[1m'
+    ENDC     = '\u001b[0m'
+
+
+class Notification(str, Enum):
+    """An Enum for notification levels.
+    """
+    OK      = "OK"
+    INFO    = "INFO"
+    WARNING = "WARNING"
+    ERROR   = "ERROR"
+
+
+def print_with_color(message: str, level: Notification) -> None:
+    """Print the message with a color for the corresponding level.
+    """
+    print(
+        BColors[level]
+        + BColors.BOLD
+        + f"{level}: [Policy]"
+        + message
+        + BColors.ENDC
+    )
+
+
+def check_tag(title):
+    _tags = ["chore", "docs", "style", "feat", "fix", "refactor", "test"]
```

Comment:
Update this tags following our commit tag formats.

Reply:
Do you mean this tags -:> https://www.notion.so/pyler/Github-ea6117eacf6940d2b89044f78209b44d#f3b30289cf554a84b554af67b4370d3b